src/main/kotlin/app/filesystem/ProfileConfig.kt

```diff
@@ -0,0 +1,47 @@
+package app.filesystem
+
+import java.util.*
+import data.model.User
+import java.io.File
+import filesystem.IDirectoryProvider
+
+// The User's Profile: the audio hash info in one file and an image in another file
+// Key for DirectoryProvider should be "Application_Name"
+class ProfileConfig(val directoryProvider: IDirectoryProvider) {
+    val locale = Locale.getDefault()
+
+    // Creates a user subdirectory and stores their hash info
+    fun storeAudio(userMine: User, audioFile: File): String {
```

Comment:
natural language is important in naming conventions. UserMine is confusing since it's not something we'd say. CurrentUser? This also carries some connotations about state. Like, it matters that the user is currently logged in, or the method has some side effects because of the current logged in user. If that's not the case, just simply user is a better name here.